compat/beaver-builder/sowb-beaver-builder.js

```diff
@@ -31,7 +31,7 @@ var sowb = window.sowb || {};
 			// Loop through the form data.
 			for ( i = 0; i < data.length; i++ ) {
 				
-				value = data[ i ].value.replace( /\r/gm, '' );
+				value = data[ i ].value.replace( /\r/gm, '' ).replace( /&#39;/g, "'" );
```

Comment:
The whole function is a direct copy of BB's `FLBuilder._getSettings` function, which is slightly modified to retrieve the values from our widgets correctly. So this code is from the latest version of BB. Should we be commenting their code?

We've since added functions in our code, which could make all this unnecessary, but in the interests of not investing too much time here, I just did the minimum necessary to fix compatibility.